src/popup/components/MultisigVaultCreateProgress.vue

```diff
@@ -132,6 +145,14 @@ export default defineComponent({
       return index === currentPhaseIndex.value + 1;
     }
 
+    function currentPhaseText(index: number, localPhase: PhaseLabel) {
```

Comment:
In my opinion this function is not necessary as this logic would still makes sense on the template side:

```
<div v-if="isPhaseCurrent(index)">
  {{ localPhase.pendingText || localPhase.text }}&hellip;
</div>
<div v-else>
  {{ localPhase.text }}
</div>
```
What do you think?

Reply:
of course, I think that's just a cosmetic change - I prefer less elements in the template